black-mulberry-core/src/main/java/com/black/mulberry/core/mapper/base/BaseMapper.java

```diff
@@ -1,5 +1,8 @@
 package com.black.mulberry.core.mapper.base;
 
+import com.black.mulberry.core.entity.User;
+import com.black.mulberry.data.transfer.model.CreateUserDto;
+
```

Comment:
Be attentive, not to add unnecessary imports

Reply:
which one is bad import?

Reply:
էս 2 impoert-ները չի օգտագործվի․ եթե չի օգտագործվի ճիշտ չէ թողնելը